src/App.js

```diff
@@ -8,15 +8,55 @@ class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies,
+      movies: [],
       clickedMovie: null,
+      isLoading: false,
+      error: '',
     }
   }
 
-  displayMovieDetails = (id) => {
-    const singleMovie = this.state.movies.find(movie => movie.id === id)
+  componentDidMount = () => {
+    this.setState({isLoading: true})
+
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+      .then(res => {
+        if (res.ok) {
+          return res.json()
+        } else {
+          console.log("Error")
+        }
```

Comment:
Nice job checking that the response is okay before parsing it.  Make sure to **throw a new Error** though so it hits the catch as opposed to logging 'Error'.